### ISSUE TITLE
Medallia - Flip Submit & Close buttons

### ISSUE THREAD

**Issue by aubreyarcangel** [department-of-veterans-affairs/va.gov-team#82137]
## Issue Description
Both the CAIA accessability audit and VA 508 office have pointed have identified the Submit and Close buttons on the Medallia modal visually does not tab order.

---
## Tasks
- [ ] Flip submit and close buttons

## Acceptance Criteria
 - [ ] **Scenario: Flipping Submit and Close buttons**
  - Given the Medallia modal has Submit and Close buttons
  - When the buttons are flipped to follow the tab order
  - Then the buttons should now follow the correct tab order
- [ ] **Scenario: Confirming accessibility compliance**
  - Given the CAIA accessability audit and VA 508 office have identified an issue
  - When the Submit and Close buttons are flipped to follow the tab order
  - Then the issue should be resolved, and the modal should be compliant with accessibility guidelines
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

## Acceptance Criteria
> Generated by [Zenhub AI](https://www.zenhub.com/ai)

**Comment by ATMiddleton** [comment]
The decision to flip the buttons has been reconsidered and the buttons will remain as is.
